pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -273,6 +273,23 @@ def test_to_json(self):
         expected = OrderedDict([('schema', schema), ('data', data)])
         assert result == expected
 
+    @pytest.mark.parametrize('test_series, expected', [(
+        pd.Series([1, 2, 3], name='a'),
+        '[{"a":1},{"a":2},{"a":3}]')
+    ])
+    def test_to_json_orient_record_with_name(self,
```

Comment:
So the point of parametrization is to "reuse" a given test case with slightly differing sets of parameters to test varying features. In this case you could get away with one test case parametrized for a name of `[None, 'a']` to test both of these cases. See if you can get that to work

Reply:
thanks for the tip!